calm-integrations/generate_task_library_items/generate_task_library_items.sh

```diff
@@ -1,7 +1,39 @@
 #!/bin/bash
 
+##############################################
+# Usage: bash bash generate_task_library_items.sh
+##############################################
+
 export PYTHONWARNINGS="ignore:Unverified HTTPS request"
+IFS=$(echo -en "\n\b")
```

Comment:
because of this IFS character n is getting truncated and parameter positions are changing too. when did `echo $pc_ip $pc_user $pc_password $pc_project` it printed `10.44.19.70 admi Nuta ix.123 r gr ssio `

```
(uitestenv) alekh.neema@C02W91BPHV2R generate_task_library_items % sh generate_task_library_items.sh
Enter Prism Central IP: 10.44.19.70
Enter Prism Central User: admin
Enter Prism Central Password: Enter Prism Central Project: regression
10.44.19.70 admi Nuta ix.123 r gr ssio
(uitestenv) alekh.neema@C02W91BPHV2R generate_task_library_items %```

Reply:
Using bash it works. Let me check